src/server/acl/user.cc

```diff
@@ -130,4 +134,22 @@ std::string_view User::Password() const {
   return password_hash_ ? *password_hash_ : default_pass;
 }
 
+const AclKeys& User::Keys() const {
+  return keys_;
+}
+
+void User::SetKeyGlobs(std::vector<UpdateKey>&& keys) {
```

Comment:
i prefer avoiding using `&&` is possible usign a simpler interface. In this case defined a value `std::vector<UpdateKey>&& keys` has the same effect, no?